activitysim/abm/models/trip_scheduling_choice.py

```diff
@@ -81,6 +82,7 @@ def generate_schedule_alternatives(tours):
 
     schedules = pd.concat([no_stops, one_way, two_way], sort=True)
     schedules[SCHEDULE_ID] = np.arange(1, schedules.shape[0] + 1)
+    schedules.sort_values(by=["tour_id", SCHEDULE_ID], inplace=True)
```

Comment:
This line is the actual "fix" for the non-reproducibility problem... let's put a comment noting that for posterity.